lnrpc/walletrpc/walletkit_server.go

```diff
@@ -681,13 +683,39 @@ func (w *WalletKit) SendOutputs(ctx context.Context,
 		return nil, err
 	}
 
+	// Before sending out funds we need to ensure that the remainder of our
+	// wallet funds would cover for the anchor reserve requirement. We'll
+	// also take unconfirmed funds into account.
+	walletBalance, err := w.cfg.Wallet.ConfirmedBalance(
+		0, lnwallet.DefaultAccountName,
+	)
+	if err != nil {
+		return nil, err
+	}
+
+	// We'll get the currently required reserve amount.
+	reserve, err := w.RequiredReserve(
+		ctx, &RequiredReserveRequest{AdditionalPublicChannels: 0},
```

Comment:
hmm what if we need more public channels?

Reply:
The `AdditionalPublicChannels` is just an RPC parameter that allows you to plan ahead if you intend to open more channels in the near future. Not really related to this feature, as we just need to check the reserve for the current number of channels, so just remove that parameter as the default is 0 anyway?

Reply:
Cool got it, yeah make sense to use the default.